src/conversions.jl

```diff
@@ -1,9 +1,30 @@
 import Convertible: findpath, haspath
 
-export rescale
+include("constants.jl")
+export rescale,
+       taitt
```

Comment:
I'm not sure we want to export the conversion functions.  If you're worried about how to call it in the test, you can just qualify it: `AstronomicalTime.taitt`.  It's used just a couple of times.

Reply:
I actually tried using like you mentioned here, `AstronomicalTime.taitt` but without import, it is raising an undefined error.

Reply:
@giordano I resolved it. Imported the function using `AstronomicalTime.Epochs.taitt`. Please review the latest commit.